slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -250,38 +253,41 @@ async def receive_messages(self) -> None:
                     break
                 try:
                     message: WSMessage = await session.receive()
-                    if self.trace_enabled and self.logger.level <= logging.DEBUG:
```

Comment:
The main purpose of this PR is to remove `trace_enabled` check here. In the built-in client, message data logging does not require `trace_enabled`. This change makes the behavior more consistent.